src/app/versioning/version.service.ts

```diff
@@ -40,7 +35,7 @@ export class VersionService {
                 this.checkVersions();
             }
 
-            IntervalObservable.create(interval).subscribe(_ => {
+            interval(delay).subscribe(_ => {
```

Comment:
This looks like a legacy code. For safety, please check if we need to call unsubscribe or not.